### PR TITLE
684 precourse faq update

### LINKED ISSUE #684: Make estimated pre course time commitments consistent
On the [About Us](http://www.makersacademy.com/about-us/) page, it says that the pre course requires 15-20 hours a week.  On the [curriculum](http://www.makersacademy.com/curriculum/) page it says it requires a minimum of 20 hours a week.  On the [pre course introduction](https://github.com/makersacademy/pre_course/blob/master/introduction.md) page, it says it requires 20-30 hours a week.  Could this be standardised to 20-30 hours a week?

### PR DESCRIPTION
**Pull request by nikeshashar**
Updating a bunch of FAQ and closes #684

### CODE REVIEW

**Comment by maryrosecook** [comment]
Thanks! There's another place to update, I think.  In the "Is the PreCourse done at home?" question, it says "You need around 20 hours per week".  Could this be changed to "You need at least 20 hours per week"?

**Comment by nikeshashar** [comment]
@maryrosecook Done

**Comment by maryrosecook** [comment]
"Our last day in the office is 16 September 2016." Is this a typo?

File: data/faqs.yml [hunk 1]
```diff
@@ -222,10 +224,36 @@
...
+
+    - question: Can I finance my deposit?
+      answer: "No. You will need to pay for your course deposit before being able to secure finance with any of our partners. We have preferential finance options in place for students of Makers Academy, so you will need to have a secured place on the course before we can confirm your registration to PCDL and Future Finance. "
+
+- category_name: Makers Academy PreCourse
+  faqs:
+    - question: How long is the PreCourse
+      answer: "The PreCourse runs for 4 weeks prior to the start of the full time course."
+
+    - question: Do students have to do a preliminary study before starting the course?
```
Comment (maryrosecook) [inline data/faqs.yml]: Delete "a"?

File: data/faqs.yml [hunk 2]
```diff
@@ -222,10 +224,36 @@
...
+      answer: "While books are kind of outdated in the fast-paced world of software development, there are a certain few that we hold dear to our heart."
 
     - question: Can I pay in Bitcoin, if so how?
       answer: "Yes - we welcome payment in Bitcoin. If you want to do so, we can talk about how to do so when you come for an interview."
 
     - question: Is there any chance to get any type of scholarship?
       answer: "We offer scholarships from time to time, usually in partnership with another company. By far the best way to hear about these things is by signing up to our newsletter and keeping an eye out for any new opportunities."
+
+    - question: Can I finance my deposit?
+      answer: "No. You will need to pay for your course deposit before being able to secure finance with any of our partners. We have preferential finance options in place for students of Makers Academy, so you will need to have a secured place on the course before we can confirm your registration to PCDL and Future Finance. "
```
Comment (maryrosecook) [inline data/faqs.yml]: For clarity, replace "so you will need to have a secured place on the course before we can confirm..." with "so you will need to have secured your place on the course by paying your deposit before we can confirm..."

File: data/faqs.yml (hunk 1 above)
Comment (nikeshashar) [inline data/faqs.yml]: wow - that's been there for years :)

Comment (nikeshashar) [inline data/faqs.yml]: changed to any

**Comment by jordanpoulton** [comment]
:shipit:

**Comment by maryrosecook** [comment]
:shipit:

**Pull request merged**